WebRoot/css/zm.css

```diff
@@ -1667,7 +1667,31 @@ div.CompDragArea * {
 
 .ZmGrantRightsDialog .addrBubbleContainer,
 .ZmPreferencesPage .addrBubbleContainer {
-    width: 290px;
+	@GrantRightsDialogInputContainer@
+}
+
+.ZmGrantRightsDialog .WindowOuterContainer {
+	@GrantRightsDialogOuterContainer@
+}
+
+.ZmGrantRightsDialog #ZmGrantRightsDialog_name_row .Label {
+	@GrantRightsDialogEmailLabel@
+}
+
+.ZmGrantRightsDialog INPUT[type='text'] {
+	@GrantRightsDialogInput@
+}
+
+.ZmGrantRightsDialog .Label:empty {
```

Comment:
".ZmGrantRightsDialog .Label:empty " selector can be removed if we are removing the empty .Label element.